.env.example

```diff
@@ -1 +1,4 @@
 DB_URL=dev mode defaults to sqlite (so this is not needed)
+AUTH0_CLIENT_ID=YOUR_CLIENT_ID
+AUTH0_SECRET=YOU_CLIENT_SECRET
```

Comment:
Minor typo on 'YOU_CLIENT_SECRET', did you mean 'YOUR_CLIENT_SECRET'. Reviewing if it has affected anything. Thanks for waiting!

Reply:
I did thanks for the catch!

Reply:
It would not affect anything, but it would still be nicer =D

Reply:
Reviewers need to create their own Auth0 account and use their account to test it locally.  @isabellberzsenyi would you be able to give reviewers directions for that?

Reply:
Updated the PR info with how to test